client/js/components/procedure/StatementSegmentsList/StatementSegment.vue

```diff
@@ -131,7 +131,7 @@
                   :text="Translator.trans('segment.oracle.tooltip')" />
                 <dp-badge
                   v-if="activeId === 'oracleRec'"
-                  class="absolute u-right-0 u-mr-0_75"
+                  class="absolute u-right-0 mr-12"
```

Comment:
wait. If its positioned absolute. Should'nt we use `r-12` instead and remove `u-right-0`?

Reply:
Yes, you're right, it's the better solution. I open a new PR.